rust/otap-dataflow/benchmarks/benches/config/main.rs

```diff
@@ -0,0 +1,235 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! Benchmark tests for config implementations
+
+use criterion::{
+    BatchSize, BenchmarkGroup, Criterion, criterion_group, criterion_main, measurement::WallTime,
+};
+
+use otap_df_config::{PipelineDag, NodeKind, SignalType};
+
+#[allow(missing_docs)]
```

Comment:
Ran into issue with missing comments on the criterion_group! macro... is there a better fix than this?